lib/routes/data/styles.js

```diff
@@ -63,4 +63,21 @@ module.exports = {
       ],
     ],
   },
+  breadcrumb: {
+    item: [
```

Comment:
Agreed with Scott. As there is no secondary data in breadcrumbs, the actual data model should be an array, not an object with one key that's an array.